src/FirstSteps/RPG/Heroes/Human.cs

```diff
@@ -6,18 +6,19 @@ namespace FirstSteps.RPG.Heroes
 {
     public class Human : Hero
     {
-        public Human(string name) : this(name, 3, 2, 5, 20, 6, 0, new Equipment())
+        public Human(string name) : this(name, 3, 2, 5, 20,20, 6, 0, new Equipment())
         {
 
         }
-        private Human(string name, int strength, int intelligence, int agility, int health, int damage, int coinsBag, Equipment equipment) : base(name, Races.Dwarf)
+        private Human(string name, int strength, int intelligence, int agility, int health,int maxHealth, int damage, int coinsBag, Equipment equipment) : base(name, Races.Human)
         {
             _strength = strength;
             _intelligence = intelligence;
             _agility = agility;
             _health = health;
             _damage = damage;
             _coinsBag = coinsBag;
+            _maxHealth = maxHealth;
```

Comment:
You don't need an extra parameter `maxHealth`. The `_maxHealth` is the same value as health's initial value.